packages/block-editor/src/hooks/padding.js

```diff
@@ -82,7 +82,7 @@ export function PaddingEdit( props ) {
 	const units = useCustomUnits( CSS_UNITS );
 	const sides = useCustomSides( blockName, 'padding' );
 
-	if ( ! hasPaddingSupport( blockName ) ) {
+	if ( useIsPaddingDisabled( { name: blockName } ) ) {
```

Comment:
Same as margin.js, you could just pass through the props here. Maybe that is a little easier to read?

```suggestion
	if ( useIsPaddingDisabled( props ) ) {
```